app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,39 @@
+<%= render 'partials/navigation' %>
+<%= stylesheet_link_tag "application", "data-turbo-track": "reload" %> 
+
+
+<% if notice%>
+  <h4 class='signin-alert'><%= notice%></h4>
+<%end%>
+<h1 id='title'>PUBLIC RECIPES</h1>
+<section class='recip-list'>
+<% @public_recipes.each do |recipe| %>
+
+  <div class="card">
+ 
+  <div class="container">
+    <section>
+      <h1><b><%= recipe.name%></b></h1> 
+    </section>
+
+    <section>
+      <p><b>Total Food Item:</b>  <%= recipe.foods.count%></p> 
+
+       <% cost = recipe.foods.map do |recipe_food| %>
+        <% recipe_food.price * recipe_food.quantity %>
+      <% end %>
+      <p><b>Total Price:</b> <%= "$#{cost.sum}" %> </p> 
+      
+    </section>
+  </div>
+ 
```

Comment:
- [ ] Kindly make sure, on the `Public recipe` page, when the user clicks on a recipe item should lead him to recipe details that match this **[wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/recipe-app/images/recipe_food.png?token=GHSAT0AAAAAABXY7ZMIDLRDXI2NKI5U2SHSY3Y554Q)**